api_tests/providers/registrations/views/test_registration_provider_moderator_detail.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    RegistrationProviderFactory,
+)
+from api_tests.providers.preprints.views.test_preprint_provider_moderator_detail import TestPreprintProviderModeratorDetail
+
+
+@pytest.mark.django_db
+class TestRegistrationProviderModeratorDetail(TestPreprintProviderModeratorDetail):
```

Comment:
That's a good point and I'll change it. Similar patterns in other tests have led to things breaking for me in the past so I definitely want to avoid that pattern.